autotest/test_gwf_newton_under_relaxation.py

```diff
@@ -0,0 +1,144 @@
+import os
+
+import flopy
+import numpy as np
+import pathlib as pl
+import pytest
+from framework import TestFramework
+from simulation import TestSimulation
+
+ex = ["nr_ur01", "nr_ur02"]
+nper = 1
+nlay = 1
+nrow = 51
+ncol = 51
+xlen = 100.0
+ylen = 100.0
+top = 25.0
+k11 = 1.0
+H1 = 7.5
+H2 = 2.5
+delr = xlen / float(ncol)
+delc = ylen / float(nrow)
+extents = (0, xlen, 0, ylen)
+shape2d = (nrow, ncol)
+shape3d = (nlay, nrow, ncol)
+nouter = 50
+ninner = 100
+hclose = 1e-8
+hclose_outer = hclose * 10.
+rclose = 1e-3
+botm = np.loadtxt(pl.Path("data/ex-gwf-bump/bottom.txt")).reshape(shape3d)
```

Comment:
This may be fine, but I think we have a `project_root_path` approach for accessing data paths.  This is from the prudic example:

```
import os
import sys

import flopy
import numpy as np
import pytest
from conftest import project_root_path
from framework import TestFramework
from simulation import TestSimulation

ex = ["prudic2004t2"]
data_path = project_root_path / "autotest" / "data"
model_path = data_path / "prudic2004test2"
fname = str(model_path / "lakibd.dat")
lakibd = np.loadtxt(fname, dtype=int)
```